README.md

```diff
@@ -129,6 +129,24 @@ Pipecat makes use of WebRTC VAD by default when using a WebRTC transport layer.
 pip install pipecat-ai[silero]
 ```
 
+## Running the Krisp Audio Filter
+
+To use the Krisp Filter in this project, you’ll need access to the **Krisp C++ SDK**.
+
+### Step 1: Obtain Access to the Krisp SDK
+1. **Create a Krisp Account**: If you don’t already have an account, [sign up at Krisp](https://krisp.ai/) to access the SDK.
+2. **Download the SDK**: Once you have an account, follow the instructions on the Krisp platform to download the [Krisp's desktop SDKs](https://sdk.krisp.ai/sdk/desktop).
+3. **Export the path to you krisp SDK**:
+`export KRISP_SDK_PATH=/PATH/TO/KRISP/SDK`
+
+### Step 2: Install the `pipecat-krisp` Module
```

Comment:
Any chance the project can be called `pipecat-ai-krisp`? Just to match `pipecat-ai`. There's another `pipecat` project completely unrelated.

Reply:
The project I mean the pypi package. The repo is fine.

Reply:
Oh, I see it's already `pipecat-ai-krisp`, then I guess just updating this text. :-D